### DISCUSSION TITLE
How to install old version of PyBaMM, say v22.9?

### DISCUSSION THREAD

**Discussion by ganmad** [pybamm-team/PyBaMM#3545] [Q&A] [answered]
Hi,

I am curious how to install old versions of PyBaMM? Thank you.

**Comment by TomTranter** [comment, marked as answer]
https://ioflood.com/blog/pip-install-specific-version/#:~:text=To%20install%20a%20specific%20version%20of%20a%20package%20using%20pip,pip%20install%20sample%3D%3D1.0.

**Reply by brosaplanella** [reply]
Note also that the requirements are not given upper bounds, so you might encounter conflicts with the dependencies.
